themes/article/app/components/Downloads.tsx

```diff
@@ -28,9 +32,10 @@ export function Downloads() {
             key={action.url}
             href={action.url}
             className="inline-block mr-2 no-underline lg:mr-0 lg:block"
+            target={action.internal ? '_self' : '_blank'}
```

Comment:
There's probably a better way to do this...?

Reply:
This also isn't actually respected for relative urls.

Reply:
changed this up significantly